encoding/encoding.go

```diff
@@ -0,0 +1,36 @@
+package encoding
+
+import (
+	"bytes"
+	"encoding/gob"
+
+	"github.com/Layr-Labs/eigenda/pkg/kzg/bn254"
+)
+
+// Proof is the multireveal proof
+// Coeffs is identical to input data converted into Fr element
+type Frame struct {
```

Comment:
nit: let's add a comment for `Frame` and move those for `Proof` and `Coeffs` to right before their definitions

Reply:
I've updated this is the followup PR.